.eslintrc

```diff
@@ -1,14 +1,13 @@
 {
   "parserOptions": {
-    "ecmaVersion": 6,
     "sourceType": "module",
     "ecmaFeatures": {
       "impliedStrict": true
-    },
+    }
   },
   "env": {
     "node": true,
-    "es6": true,
+    "es2020": true
```

Comment:
es2020 is still not supported in Node 10. I hesitate to add linting support since it might lead to us introducing features that depend on newer JS that is not yet supported in our runtime.

https://node.green/#ES2020

Reply:
so we have to hold off until we deprecate stapler ?

Reply:
Until we upgrade Node to a version that supports it. Looks like Node 12 has decent support.

Reply:
@mattolson, @junedkazi 
Actually, Node will just break when encounter unsupported syntax, so if we have good test coverage - we won’t miss these parts.
But I can just set "es2018" for the linter, it seems to have good support by Node 10